docs/conf.py

```diff
@@ -196,7 +196,11 @@
 .. raw:: html
 
     <div>
-      <p><a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download this notebook</a></p>
+      <p><a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download this notebook</a>
+      <br>Run the master version of this notebook:
```

Comment:
Also, it's not directly related to this PR, but I realised that "master version" probably doesn't mean much to a user, an alternative would be something like:

```suggestion
      <br>Run the latest release of this notebook:
```

Future work along this line would be changing it to "Run this notebook", by using something like nbsphinx approach. They use `env.config.release` to have each doc version page execute with exactly that git tag https://github.com/spatialaudio/nbsphinx/blob/0.6.1/doc/conf.py#L51 .

Reply:
"latest release" does read better - have adopted in the notebooks themselves; in sphinx I've adopted your suggestion below.